From 254f13ebf863c9884b9b229a91697e5f7c791a26 Mon Sep 17 00:00:00 2001
From: vivekana <vivekana@xilinx.com>
Date: Wed, 7 Jul 2021 18:49:04 -0700
Subject: [PATCH] Jupyterlab: Serve notebooks form user defined dir

Jupyter configuration failed to read form notebook config file.
clean up configs already part of dameon args
---
 .../start-jupyterlab/start-jupyterlab/jupyter_lab_config.py         | 3 +++
 .../start-jupyterlab/start-jupyterlab/jupyter_notebook_config.py    | 6 ------
 2 files changed, 3 insertions(+), 6 deletions(-)
 create mode 100644 recipes-utils/start-jupyterlab/start-jupyterlab/jupyter_lab_config.py
 delete mode 100644 recipes-utils/start-jupyterlab/start-jupyterlab/jupyter_notebook_config.py

diff --git a/jupyter_lab_config.py b/jupyter_lab_config.py
new file mode 100644
index 0000000..feb0190
--- /dev/null
+++ b/jupyter_lab_config.py
@@ -0,0 +1,3 @@
+c.ServerApp.root_dir = '/usr/share/notebooks'
+c.ServerApp.port = 8888
+c.ServerApp.iopub_data_rate_limit = 100000000
diff --git a/jupyter_notebook_config.py b/jupyter_notebook_config.py
deleted file mode 100644
index aff1bdd..0000000
--- a/jupyter_notebook_config.py
+++ /dev/null
@@ -1,6 +0,0 @@
-c.NotebookApp.notebook_dir = '/opt/xilinx/share/notebooks'
-c.NotebookApp.port = 8888
-c.NotebookApp.iopub_data_rate_limit = 100000000
-c.NotebookApp.allow_root=True
-c.NotebookApp.open_browser=False
-
-- 
2.7.4

